workflow/rocoto_funcs/jedivar.py

```diff
@@ -112,11 +114,37 @@ def jedivar(xmlFile, expdir, do_spinup=False):
         iodadep = f'<datadep age="00:01:00"><cyclestr>&COMROOT;/&NET;/&rrfs_ver;/&RUN;.@Y@m@d/@H/ioda_bufr/det/ioda_aircar.nc</cyclestr></datadep>'
 
     #
+    coldhrs = coldhrs.split(' ')
+    strneqs = ""
+    streqs = ""
+    if coldstart_cyc_do_da.upper() == "FALSE":
+        spaces = " " * 4
+        strneqs = '<or>'
+        streqs = '<or>'
+        for hr in coldhrs:
+            hr = f"{hr:0>2}"
+            strneqs += '\n' + spaces + f'  <strneq><left><cyclestr>@H</cyclestr></left><right>{hr}</right></strneq>'
+            streqs += '\n' + spaces + f'  <streq><left><cyclestr>@H</cyclestr></left><right>{hr}</right></streq>'
+        strneqs += '\n' + spaces + '</or>'
+        streqs += '\n' + spaces + '</or>'
+        da_dep = f'''<or>
+    <and>
+    {strneqs}
+    {iodadep}{ens_dep}
+    </and>
+    {streqs}
+    </or>
+        '''
+    else:
+        da_dep = f'''
+        {iodadep}{ens_dep}
+        '''
+    #
```

Comment:
@chunhuazhou Could you elaborate the issue you met for the changes here? Thanks!

Reply:
When we have COLDSTART_CYCS_DO_DA=false, the dependency on the ensemble forecast files at cold start cycles are not needed and would slow down the whole cycling. The changes here address this in different situations:
- if COLDSTART_CYCS_DO_DA=false, disable ensemble data dependency for the jedivar task at cold start cycles; keep the ensemble data dependency for the other cycles
-  if COLDSTART_CYCS_DO_DA=true, keep the ensemble data dependency for all the cycles

Reply:
Looks great. Thank you!